packages/lib/src/__tests__/api-command.test.ts

```diff
@@ -72,6 +90,93 @@ describe('api-command', () => {
 			expect(configUsed?.headers).toEqual({ 'Accept-Language': 'es-US' })
 		})
 
+		it('passes organization flag on to client', async () => {
+			await apiCommand.setup({}, [], { organization: 'organization-id-from-flag' })
+			const stClientSpy = jest.spyOn(coreSDK, 'SmartThingsClient')
+
+			expect(stClientSpy).toHaveBeenCalledTimes(1)
+
+			const configUsed = stClientSpy.mock.calls[0][1]
+			expect(configUsed?.headers).toEqual({ 'X-ST-Organization': 'organization-id-from-flag' })
+		})
+
+		it('passes organization config on to client', async () => {
+			jest.spyOn(CLIConfig.prototype, 'getProfile')
+				.mockImplementationOnce(() => ({ organization: 'organization-id-from-config' }))
+
+			await apiCommand.setup({}, [], {})
+
+			const stClientSpy = jest.spyOn(coreSDK, 'SmartThingsClient')
+
+			expect(stClientSpy).toHaveBeenCalledTimes(1)
+
+			const configUsed = stClientSpy.mock.calls[0][1]
+			expect(configUsed?.headers).toEqual({ 'X-ST-Organization': 'organization-id-from-config' })
+		})
+
+		it('prefers organization flag over config', async () => {
+			jest.spyOn(CLIConfig.prototype, 'getProfile')
+				.mockImplementationOnce(() => ({ organization: 'organization-id-from-config' }))
+
+			await apiCommand.setup({}, [], { organization: 'organization-id-from-flag' })
+
+			const stClientSpy = jest.spyOn(coreSDK, 'SmartThingsClient')
+
+			expect(stClientSpy).toHaveBeenCalledTimes(1)
+
+			const configUsed = stClientSpy.mock.calls[0][1]
+			expect(configUsed?.headers).toEqual({ 'X-ST-Organization': 'organization-id-from-flag' })
+		})
+
+		describe('warningLogger', () => {
+			it('uses string as-is', async () => {
+				await apiCommand.setup({}, [], { language: 'es-US' })
+				const stClientSpy = jest.spyOn(coreSDK, 'SmartThingsClient')
+
+				expect(stClientSpy).toHaveBeenCalledTimes(1)
+
+				const configUsed = stClientSpy.mock.calls[0][1]
+				const warningLogger = configUsed?.warningLogger
+				expect(warningLogger).toBeDefined()
+				if (!warningLogger) {
+					throw Error('warningLogger must be defined')
+				}
```

Comment:
I don't think this is required (also in the [test below](https://github.com/SmartThingsCommunity/smartthings-cli/pull/253/files#diff-1a665b1bd5ba0ae39159234b9433ebdfa250d2ffb7a351ed4782b0bcbe5b4976R161)).

Reply:
Oops, I meant to fix this after I learned about it from the other review and forgot. It's fixed now.